### ISSUE TITLE
[BUG] SkiaSharpGenerator fails 

### ISSUE THREAD

**Issue by dr-vij** [mono/SkiaSharp#1852]
Generator/tests and etc seems do not work:

vij@Viktors-MacBook-Air SkiaSharp % dotnet run --project=utils/SkiaSharpGenerator/SkiaSharpGenerator.csproj -- generate --config binding/libSkiaSharp.json --skia externals/skia --output binding/Binding/SkiaApi.generated.cs

Starting C# API generation...
ERROR: Parsing headers failed.
ERROR: 'stdint.h' file not found at /Users/vij/WorkProjects/ThirdParty/SkiaSharp/externals/skia/./include/c/sk_types.h(13, 10)
ERROR: skiasharp-tools: An error occurred: `One or more errors occurred. (Parsing headers failed.)`
vij@Viktors-MacBook-Air SkiaSharp %

**Comment by mattleibow** [comment]
Yeah, I think I only got it to work on windows. You also need vs installed I think. I never actually used it on mac and just switch to my Windows box for this step.

I think I just need to configure the package I am using with the paths to the clang compiler...

**Comment by dr-vij** [comment]
Thank you! I will try from virtual machine with Windows onboard! :)
